src/Sylius/Behat/Page/Shop/Product/ShowPage.php

```diff
@@ -134,7 +134,7 @@ public function hasAddToCartButton(): bool
             return false;
         }
 
-        return $this->getElement('add_to_cart_button') !== null && false === $this->getElement('add_to_cart_button')->hasAttribute('disabled');
+        return false === $this->getElement('add_to_cart_button')->hasAttribute('disabled');
```

Comment:
```suggestion
        return !$this->getElement('add_to_cart_button')->hasAttribute('disabled');
```

Reply:
If only `hasAttribute` contract supports only boolean, I'm in favor of the proposed change.